seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/util/ClickhouseCatalogUtil.java

```diff
@@ -36,7 +36,14 @@ public String columnToConnectorType(Column column) {
         if (column.getSinkType() != null) {
             columnType = column.getSinkType();
         } else {
-            columnType = ClickhouseTypeConverter.INSTANCE.reconvert(column).getColumnType();
+            if (column.isNullable()) {
+                columnType =
+                        "Nullable("
+                                + ClickhouseTypeConverter.INSTANCE.reconvert(column).getColumnType()
+                                + ")";
+            } else {
+                columnType = ClickhouseTypeConverter.INSTANCE.reconvert(column).getColumnType();
+            }
```

Comment:
`columnType = column.isNullable() ? "Nullable(" + reconvertedColumnType + ")" : reconvertedColumnType;`
This way of writing should be more concise